rasa/core/actions/forms.py

```diff
@@ -551,78 +551,6 @@ def _should_request_slot(tracker: "DialogueStateTracker", slot_name: Text) -> bo
         """Check whether form action should request given slot."""
         return tracker.get_slot(slot_name) is None
 
-    async def activate(
```

Comment:
This is a key public method and shouldn't be removed as it's classified as a breaking change.